app/models/instance_note.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: instance_notes
+#
+#  id         :bigint(8)        not null, primary key
+#  domain     :string
+#  account_id :bigint(8)
+#  content    :text
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+class InstanceNote < ApplicationRecord
+  include DomainNormalizable
+  include DomainMaterializable
+
+  CONTENT_SIZE_LIMIT = 2_000
+
+  belongs_to :account
+  belongs_to :instance, inverse_of: :notes, foreign_key: :domain, primary_key: :domain, optional: true
```

Comment:
The `optional: true` here allows us to create an instance note for an instance that doesn't yet exist in the materialised view for `Instances`

Reply:
Note: We probably want to update the `Instances` view to surface instances that have notes. I'm not sure how to do such a migration though.

cc @ClearlyClaire